crates/ruff/src/settings/types.rs

```diff
@@ -101,9 +101,10 @@ pub enum PreviewMode {
 
 impl From<bool> for PreviewMode {
     fn from(version: bool) -> Self {
-        match version {
-            true => PreviewMode::Enabled,
-            false => PreviewMode::Disabled,
+        if version {
+            PreviewMode::Enabled
+        } else {
+            PreviewMode::Disabled
         }
```

Comment:
This always reminds me of @charliermarsh's tweet about how high-performance code looks like :laughing: